sweetsourcod/lempel_ziv.pyx

```diff
@@ -20,4 +20,12 @@ def lempel_ziv_factors(lattice, version='lz77'):
             factors = get_lz77_factors(lattice)
     else:
         raise NotImplementedError
-    return factors
\ No newline at end of file
+    return factors
+    
+
+cpdef cross_parsing_complexity(lattice1, lattice2):
```

Comment:
why is this cpdef and the other cdef?

Reply:
cross_parsing_complexity returns the complexity and the sumlog, while the other returns the direct parsing results (list of {position, length})